sdk/storage/storage-file-datalake/src/DataLakeFileSystemClient.ts

```diff
@@ -194,6 +194,12 @@ export class DataLakeFileSystemClient extends StorageClient {
         ...options,
         access: toContainerPublicAccessType(options.access),
         tracingOptions: updatedOptions.tracingOptions,
+        containerEncryptionScope: options.fileSystemEncryptionScope
+          ? {
+              defaultEncryptionScope: options.fileSystemEncryptionScope.defaultEncryptionScope,
+              preventEncryptionScopeOverride: true,
```

Comment:
>preventEncryptionScopeOverride: true, [](http://example.com/codeflow?start=14&length=37)

Need add comment to `options.fileSystemEncryptionScope`
for the default "preventEncryptionScopeOverride: true," behavior.

And do we really need to additionally add it? Why don't let the server do that on its own behalf.

Reply:
At the very beginning, .Net doesn't expose interface for preventEncryptionScopeOverride and always set it to true. Now .Net added preventEncryptionScopeOverride to interface, so we'll also add it to our interface.  File system is created by blob container creation with blob endpoint, it already supports this setting.  Only there's no interface to change encryption scope to each file/path.